src/javascript/EditPanel/EditPanelContent/FormBuilder/Section/FieldSet/Field/Field.actions.jsx

```diff
@@ -31,7 +31,7 @@ export const FieldActionsCmp = ({classes, inputContext, selectorType, field, for
         } else {
             actionCmp = (
                 <>
-                    <ContextualMenu ref={contextualMenu}
```

Comment:
Not only the name but also the content which changed. All usages of contextualMenu should be updated

Reply:
do you mean the value of setOpenRef (if yes, it's a ref as used in jcontent .. ?) ? or another attribute ?

Reply:
just mean that if it's a ref to open, you cannot call contextualMenu.current.open(), but directly contextualMenu.current(). ContextualMenu has been converted to functional component, so you cannot have a ref to it.